### ISSUE TITLE
REPL does not record mouse interactions on https://onedrive.live.com/about/en-us/signin/

### ISSUE THREAD

**Issue by sushil-relicx** [rrweb-io/rrweb#570]
Trying to record interactions with OneDrive. using npm run repl. The mouse interaction and user name entry on this page is not captured. 

Trying the same operation at https://login.live.com/login.srf works.

```
C:\Users\sushi\Downloads\rrweb-master\rrweb-master>npm run repl

> rrweb@0.9.14 repl
> npm run bundle:browser && cross-env TS_NODE_CACHE=false TS_NODE_FILES=true ts-node scripts/repl.ts


> rrweb@0.9.14 bundle:browser
> cross-env BROWSER_ONLY=true rollup --config


./src/index.ts → dist/rrweb.min.js...
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
created dist/rrweb.min.js in 4s
? Enter the url you want to record, e.g https://react-redux.realworld.io:  https://onedrive.live.com/about/en-us/signin/
```

**Comment by Mark-Fenng** [comment]
What's the link on OneDrive?

**Comment by Mark-Fenng** [comment]
<img width="1079" alt="image" src="https://user-images.githubusercontent.com/27533910/194012469-4cb99466-1197-4b66-aab8-10a01df5127c.png">
When recording on https://onedrive.live.com/about/en-us/signin/, first 10 seconds are totally empty.

**Comment by Mark-Fenng** [comment]
After checking, the cause for this issue is the page uses a cross-origin iframe to provide the login function. So this issue can be equal to how to record cross-origin IFrames.

**Comment by Juice10** [comment]
Fixed thanks to #1035
